src/transformers/modeling_utils.py

```diff
@@ -376,7 +380,11 @@ def shard_checkpoint(
             sharded_state_dicts[block_id][key] = weight
             continue
 
-        weight_size = weight.numel() * dtype_byte_size(weight.dtype)
+        if weight.__class__.__name__ in ["QTensor", "QBitsTensor"]:
+            # we don't add the scales/zero-point since these are negligeable
```

Comment:
Not so much anymore with group-wise quantization, so you might consider adding them after all (and we might want to pack zeropoint in quanto too since it is now bigger).

Reply:
Let's not add too much code related to quanto, especially if we decide to pack the zero-point or quantize the scale again in the future. I think that we might be off by one GB per file (shard_size of 5GB), so that should be fine. LMK if this sounds reasonable.